homeassistant/components/habitica/sensor.py

```diff
@@ -39,6 +41,78 @@
     "class": SensorType("Class", "mdi:sword", None, ["stats", "class"]),
 }
 
+
+@dataclass(kw_only=True, frozen=True)
+class HabitipySensorEntityDescription(SensorEntityDescription):
+    """Habitipy Sensor Description."""
+
+    value_path: list[str]
+
+
+SENSOR_DESCRIPTIONS: dict[str, HabitipySensorEntityDescription] = {
+    "name": HabitipySensorEntityDescription(
+        key="name",
+        translation_key="name",
+        value_path=["profile", "name"],
+    ),
+    "hp": HabitipySensorEntityDescription(
+        key="hp",
+        translation_key="hp",
+        native_unit_of_measurement="HP",
+        suggested_display_precision=0,
+        value_path=["stats", "hp"],
+    ),
+    "maxHealth": HabitipySensorEntityDescription(
+        key="maxHealth",
+        translation_key="maxhealth",
```

Comment:
Let keep it nice snake case


```suggestion
        translation_key="max_health",
```